### ISSUE TITLE
panic from `git lfs lfs-files`

### ISSUE THREAD

**Issue by kennethlimcp** [github/git-lfs#699]
did the following:
- `git init` 
- `git-lfs init` 
- `git lfs track *.exe`
- `git lfs ls-files`

and got this:

```
KENMBP:git-lfs-test kennethlimcp$ git lfs ls-files
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/github/git-lfs/git.ResolveRef(0x4836a0, 0x4, 0x1, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/git/git.go:68 +0x2f6
github.com/github/git-lfs/git.CurrentRef(0x6cdf08, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/git/git.go:73 +0x38
github.com/github/git-lfs/commands.lsFilesCommand(0x6a46a0, 0x6cdf08, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/commands/command_ls_files.go:26 +0x205
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).execute(0x6a46a0, 0x6cdf08, 0x0, 0x0, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:477 +0x403
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).Execute(0x6a5bc0, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:551 +0x46a
github.com/github/git-lfs/commands.Run()
	/Users/rick/go/src/github.com/github/git-lfs/commands/commands.go:85 +0x23
main.main()
	/Users/rick/go/src/github.com/github/git-lfs/git-lfs.go:9 +0x14

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1
```

I think this happens when you did not commit the file first or something along that line that caused the issue

**Comment by technoweenie** [comment]
What version of Git LFS? Can you run `git lfs env`?

**Comment by kennethlimcp** [comment]
```
KENMBP:git-lfs-test kennethlimcp$ git lfs env
git-lfs/0.6.0 (GitHub; darwin amd64; go 1.5.1; git 183bdc8)
git version 2.3.8 (Apple Git-58)

Endpoint=https://github.com/kennethlimcp/git-lfs-test.git/info/lfs (auth=basic)
LocalWorkingDir=/Users/kennethlimcp/Desktop/git-lfs-test
LocalGitDir=/Users/kennethlimcp/Desktop/git-lfs-test/.git
LocalGitStorageDir=/Users/kennethlimcp/Desktop/git-lfs-test/.git
LocalMediaDir=/Users/kennethlimcp/Desktop/git-lfs-test/.git/lfs/objects
TempDir=/Users/kennethlimcp/Desktop/git-lfs-test/.git/lfs/tmp
ConcurrentTransfers=3
BatchTransfer=true
```

**Comment by technoweenie** [comment]
Try upgrading to v1.0. This should be fixed in #668
